sdks/java/io/common/build.gradle

```diff
@@ -17,7 +17,7 @@
  */
 
 plugins { id 'org.apache.beam.module' }
-applyJavaNature(exportJavadoc: false)
+applyJavaNature(exportJavadoc: false, automaticModuleName: 'org.apache.beam.sdk.io.common')
```

Comment:
@lukecwik I think that this module should also not be published - I think I missed this. It contains only /test utilities and is used only by test code. 

Let me know if you agree and if I should notify the devlist again or simply stop publishing.